recipes/r-ca/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set version = '0.70' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-ca
+  version: {{ version|replace("-", "_") }}
+
+source:
+  fn: ca_{{ version }}.tar.gz
+  url:
+    - https://cran.r-project.org/src/contrib/ca_{{ version }}.tar.gz
+    - https://cran.r-project.org/src/contrib/Archive/ca/ca_{{ version }}.tar.gz
+
+  sha256: 34fc1b165c13d78d6605656d1c32a4192e3ea4c0d0da12c3041f2632ed61601c
+
+build:
+  number: 0
+  skip: true  # [win32]
+
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - r-base
+
+  run:
+    - r-base
+
+test:
+  commands:
+    - $R -e "library('ca')"  # [not win]
+    - "\"%R%\" -e \"library('ca')\""  # [win]
+
+about:
+  home: https://cran.r-project.org/web/packages/ca/index.html
+  license: GPL
+  summary: Computation and visualization of simple, multiple and joint correspondence analysis.
+  license_family: GPL
```

Comment:
license_file is missing here

Reply:
Hmm, don't know what to do about this one. It's now included when creating the recipe through conda skeleton cran 😕

Reply:
you can just add this one-liner with `license_file:`?

Reply:
@MathiasHaudgaard Here's how I was advised to handle the license for R packages that are ambiguous about the GPL version:

https://github.com/conda-forge/staged-recipes/pull/4676/files#diff-ab236354f787c30256ad43a827c41f18R42

```
about:
  license: GPL-2.0 and GPL-3.0
  license_family: GPL
  license_file: '{{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-3'  # [unix]
  license_file: '{{ environ["PREFIX"] }}\R\share\licenses\GPL-3'  # [win]
```